src/modules/sensors/sensor_params_battery.c

```diff
@@ -98,10 +98,20 @@ PARAM_DEFINE_FLOAT(BAT_A_PER_V, -1.0);
  * means that measurements are expected to come from a power module. If the value is set to
  * 'External' then the system expects to receive mavlink battery status messages.
  *
- * @min 0
+ * @min -1
```

Comment:
Why is this changed?

Reply:
Rolled back. I thought -1 was the default value.